### ISSUE TITLE
Timestamp Index

### ISSUE THREAD

**Issue by cbratschi** [winstonjs/winston-mongodb#55]
The query calls are getting slower and slower with more and more log items. The reason is the missing index for timestamp. I am getting the last items which means MongoDB has to process all of them to order them by date.

Could you please create an index?

Here is a code snippet:

```javascript
db.createCollection(self.collection, opts, function(err, coll) {
    if (err) {
        return cb(err);
    }

    coll.createIndex({ timestamp: -1 }, function (err) {
        cb(err, db);
    });
});
```

A couple of error conditions are silently ignored by winston-mongodb. Would be good to log them on the console at least.

Regards,
Christoph

**Comment by 39dotyt** [comment]
Thank you for noticing. This makes sense. Update is coming.

**Comment by 39dotyt** [comment]
Fixed in v1.2.0.

**Comment by cbratschi** [comment]
Great. Thanks for the quick update.
